### ISSUE TITLE
OOMs

### ISSUE THREAD

**Issue by rex-remind101** [DataDog/dd-trace-js#201]
We installed the latest version (0.5.1) and deployed and all of our instances began throwing out-of-memory errors almost immediately. Rolling back the commit (removing dd-trace) fixed the issue. Other than a change to `package.json` the following is the entirety of our diff:
```
+const tracer = require('dd-trace');
+if (process.env.DD_TRACE_ENABLED === 'true') {
+  tracer.init({ plugins: false });
+
+  tracer.use('http');
+  tracer.use('express');
+  tracer.use('graphql');
+}
+
```

**Comment by rochdev** [comment]
Possibly related to #197. I am currently investigating memory issues.

**Comment by rochdev** [comment]
Can you try with 0.5.2-beta.0?

**Comment by rochdev** [comment]
Fixed by #203

**Comment by rochdev** [comment]
Released in 0.5.2
